### PR TITLE
Add StatusChangedFrom

### PR DESCRIPTION
**Pull request by jinnnn9**

### CODE REVIEW

File: src/foam/nanos/crunch/predicate/StatusChangedFrom.js [hunk 1]
```diff
@@ -0,0 +1,60 @@
+/**
+ * @license
+ * Copyright 2023 The FOAM Authors. All Rights Reserved.
+ * http://www.apache.org/licenses/LICENSE-2.0
+ */
+
+foam.CLASS({
```
Comment (jlhughes) [inline src/foam/nanos/crunch/predicate/StatusChangedFrom.js:7]: Why can't we do this with fscript? 
n.status!=o.status 
and can be combined with checking some particular status
o.status==...&&n.status!=o.status  
for example.

Comment (jinnnn9) [reply src/foam/nanos/crunch/predicate/StatusChangedFrom.js:7]: I need it for a rule predicate. Can we use fcript in this case?

Comment (jlhughes) [reply src/foam/nanos/crunch/predicate/StatusChangedFrom.js:7]: yes, see foam.nanos.crunch.invoke-edit-behaviour for an example: 
        class: 'foam.nanos.ruler.predicate.FScriptRulePredicate',
        query: 'o.status==foam.nanos.crunch.CapabilityJunctionStatus.GRANTED'

**Comment by jinnnn9** [comment]
closing this as we can use the predicate can be replaced with FscriptRulePredicate

**Pull request closed without merging**